web/wp-content/themes/dctx2/inc/template-tags.php

```diff
@@ -472,6 +472,50 @@ function dctx_display_numeric_pagination( $args = array() ) {
 	<?php
 }
 
+
+/**
+ * Displays post pagination on single posts.
+ *
+ * @param array $args Array of params to customize output.
+ *
+ * @author JC Palmes
+ */
+function dctx_the_post_navigation( $args = array() ) {
+	$args = wp_parse_args( $args, array(
```

Comment:
:no_entry_sign: **Error**: Only one argument is allowed per line in a multi-line function call (*PEAR.Functions.FunctionCallSignature.MultipleArguments*).